content/python/concepts/built-in-functions/terms/divmod/divmod.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: 'divmod()'
+Description: 'Returns the quotient and remainder of the division of two numbers.'
+Subjects:
+
+    'Computer Science'
+    'Mathematics'
+    Tags:
+    'Functions'
+    'Math'
+    'Numbers'
+    CatalogContent:
+    'learn-python-3'
+    'paths/computer-science'
+    'paths/mathematics'
+---
+
+Returns the quotient and remainder of the division of two numbers.
+
+## Syntax
+
+```py
+divmod(x, y)
+```
+
```

Comment:
underneath the syntax there should be an explanation, even if it's very brief, such as-
The `divmod()` method takes only two arguments and returns a tuple, with the result of the division and the remainder.

Reply:
should I update the other terms that don't have an explanation?